instrumentation/sequel/test/opentelemetry/instrumentation/sequel_test.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Instrumentation::Sequel do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Sequel::Instrumentation.instance }
+  let(:exporter) { EXPORTER }
+  let(:memory_database) { Sequel.connect('sqlite::memory:') }
+  let(:db_file) { Tempfile.new }
+  let(:file_database) { Sequel.connect("sqlite://#{db_file.path}") }
+  let(:model) { Class.new(Sequel::Model(:documents)) }
+
+  before do
+    instrumentation.install
+    memory_database.extension :opentelemetry
+    memory_database.create_table!(:documents) do
+      primary_key :id
+      String :title
+    end
+    file_database.extension :opentelemetry
+    model # Instantiate the model so that it doesn't interfere with further calls
+
+    exporter.reset
+  end
+
+  describe 'database methods' do
+    it 'before query' do
+      _(exporter.finished_spans.size).must_equal 0
+    end
+
+    it 'traces dataset executes' do
+      memory_database['SELECT 1'].all
+
+      _(exporter.finished_spans.size).must_equal 1
+      _(exporter.finished_spans.first.name).must_equal('sequel.execute')
```

Comment:
Similar to the test above, you will be able to test both the size of the array and the contents at once by comparing objects instead of using size properties and navigating the objects. 

```suggestion
      _(exporter.finished_spans.map(&:name)).must_equal(['sequel.execute'])
```

I would recommend applying this change to similar tests in this file.